helpers/image-attribs.sh

```diff
@@ -27,10 +27,8 @@
 get_attr() {
     local output=""
 
-    output="$("${PYTHON}" "${PIEMAN_UTILS_DIR}"/image_attrs.py --file="${YML_FILE}" "$@" 2>&1)"
-    # TODO: check exit code directly with e.g. 'if mycmd;'
-    # shellcheck disable=SC2181
-    if [ $? -ne 0 ]; then
+    { output="$("${PYTHON}" "${PIEMAN_UTILS_DIR}"/image_attrs.py --file="${YML_FILE}" "$@" 2>&1)"; exit_code="$?"; } || true
+    if [ "${exit_code}" -ne 0 ]; then
         fatal "while getting the specified attribute from ${YML_FILE} " \
```

Comment:
Turned out that the space after `${YML_FILE}` is redundant. Could you remove that in a separate commit?